spec/support/webmock/court_data_api/webmock.rb

```diff
@@ -248,4 +248,44 @@
       body: load_json_stub('cd_api/defendants_body.json')
     )
   end
+
+  config.before(:each, stub_v2_hearing_summary: true) do
+    stub_request(
+      :get, %r{/v2/hearingsummaries/#{case_reference}}
+    ).to_return(
+      status: 200,
+      headers: { 'Content-Type' => 'application/json' },
+      body: load_json_stub('cd_api/hearing_summary_response.json')
+    )
+  end
+
+  config.before(:each, stub_v2_hearing_summary_error: true) do
+    stub_request(
+      :get, %r{/v2/hearingsummaries/#{case_reference}}
+    ).to_return(
+      status: 500,
+      body: ''
+    )
+  end
+
+  config.before(:each, stub_hearing_summaries_v2: true) do
+    stub_request(:get, %r{http.*/v2/hearingsummaries/})
+      .to_return(
+        status: 200,
+        body: load_json_stub('cd_api/hearingsummaries_body.json'),
+        headers: { 'Content-Type' => 'application/json' }
+      )
+  end
```

Comment:
Should this stub be here? The request url is incorrect

Reply:
resolved